h2o-core/src/test/java/water/parser/ParserTestARFF.java

```diff
@@ -4,21 +4,18 @@
 import org.junit.BeforeClass;
 import org.junit.Ignore;
 import org.junit.Test;
-import water.DKV;
-import water.H2O;
-import water.Key;
-import water.TestUtil;
+import water.*;
```

Comment:
Watch out for those bulk imports, I believe we had a policy about that with @michalkurka . I try not to use them. But this is not an issue for performance really. Mostly this is an issue of being unable to see new imports in new PRs.

Reply:
Tried to fix it but IntelliJ just replaced it with import water.*;